### PR TITLE
CORTX-29227: RGW: support bundle should be generated even if config file is missing

### PR DESCRIPTION
**Pull request by gargpalak048**
Signed-off-by: Palak Garg <palak.garg@seagate.com>

# Problem Statement
- According to current implementation, rgw support bundle will raise error, if config file is missing.

# Design
- support bundle should skip the config file if missing, and bundle up other log files present.

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [x] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by sachitanands — APPROVED** [review]

File: src/rgw/support/rgw_support_bundle [hunk 1]
```diff
@@ -60,11 +60,9 @@ class RGWSupportBundle:
             RGWSupportBundle._cleanup()
         os.makedirs(RGWSupportBundle._tmp_src, exist_ok=True)
         # copy configuration files
-        try:
+        if os.path.exists(config_file):
```
Comment (rahul-tripathi-git) [inline src/rgw/support/rgw_support_bundle:63]: If config file is not present, are we not logging it?

File: src/rgw/support/rgw_support_bundle [hunk 2]
```diff
@@ -60,11 +60,9 @@ class RGWSupportBundle:
...
             shutil.copyfile(config_file,\
                 os.path.join(RGWSupportBundle._tmp_src, RGW_CONF_FILE))
```
Comment (rahul-tripathi-git) [inline src/rgw/support/rgw_support_bundle:65]: If config file is not present, are we not logging it?

Comment (gargpalak048) [reply src/rgw/support/rgw_support_bundle:67]: Added Logging for rgw support bundle

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me.

**Comment by gargpalak048** [comment]
Tested with custom build: https://eos-jenkins.colo.seagate.com/job/GitHub-custom-ci-builds/job/generic/job/custom-ci/5292
Deployment is successful.

SB is generating for rgw

> tar -xvf SBag9s2g01_469f965c135a46859a9f9aff49bf5ec7.tar.gz
SBag9s2g01/
SBag9s2g01/common/
SBag9s2g01/common/common.tar.gz
SBag9s2g01/rgw/
SBag9s2g01/rgw/rgw_SBag9s2g01_469f965c135a46859a9f9aff49bf5ec7_rgw.tar.gz
SBag9s2g01/utils/
SBag9s2g01/utils/SBag9s2g01_469f965c135a46859a9f9aff49bf5ec7_utils.tar.gz
SBag9s2g01/summary.yaml

and rgw support bundle log file will log error if config file is not present

> [root@cortx-server-headless-svc-ssc-vm-rhev4-2192 SBag9s2g01]# cat rgw/rgw_support_bundle.log
2022-03-15 08:14:02 rgw_support_bundle [590]: INFO [generate] rgw support bundle generation started!!
2022-03-15 08:14:02 rgw_support_bundle [590]: ERROR [generate] /etc/cortx/rgw/469f965c135a46859a9f9aff49bf5ec7/cortx_rgw.conf is not present.Skipping the config file collection for rgw

**Review by sachitanands — APPROVED** [review]

**Pull request merged**